src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const newObject = {};
+  const clearString = sourceString.trim();
+  const separation = clearString.split(';');
+
+  separation.forEach((declaration) => {
+    const [key, value] = declaration.split(':').map((part) => part.trim());
+
+    if (key && value) {
+      newObject[key] = value;
+    }
+  });
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. Here, `forEach` is used correctly as it's an exception to the rule.